pkg/config.go

```diff
@@ -72,13 +72,19 @@ func LoadGlobalBackends() error {
 	return nil
 }
 
+var errRoutesNotProvided = fmt.Errorf("no routes provided")
+
 // getBackendsFromConfigs instantiates backends from the given configuration.
 //
 // A single configuration can have multiple backends.
 func getBackendsFromConfigs(kommonsClient *kommons.Client, backendConfig logs.SearchBackendConfig) ([]logs.SearchBackend, error) {
 	var backends []logs.SearchBackend
 
 	if backendConfig.Kubernetes != nil {
+		if len(backendConfig.Kubernetes.Routes) == 0 {
```

Comment:
I think one misconfigured backend shouldn't mean we discard all other backends.

Reply:
But this is getting backend for a unit of config that the user supplies, assume you are using all the backends in the config, do we
a) Want them all to fail explicitly
b) Silently fail (just log) bad backends but continue with ones that can work.

I'd prefer to go for A since from a user POV either a config should just work completely or just fail, no half measures

What are your opinions about this ?

Reply:
I'd prefer failing explicitly as well but the `SetupBackends` func fails silently (just logs) after all. So at the end, the program continues to run but with none of the backend actually there because of one misconfiguration.

Reply:
@adityathebe Ahh, that makes sense. I have fixed the SetupBackends flow now.

If out of multiple configs, one is misconfigured, that one will be logged but other configs from DB will continue working